pkg/conduit/runtime.go

```diff
@@ -155,13 +160,23 @@ func NewRuntime(cfg Config) (*Runtime, error) {
 		connectorPluginService: connPluginService,
 		processorPluginService: procPluginService,
 
+		schemaService: schema.NewService(),
+
 		connectorPersister: connectorPersister,
 
-		logger: logger,
+		gRPCStatsHandler: newGRPCStatsHandler(),
+		logger:           logger,
 	}
 	return r, nil
 }
 
+func newGRPCStatsHandler() *promgrpc.StatsHandler {
+	h := promgrpc.ServerStatsHandler()
+	promclient.MustRegister(h)
+
+	return h
+}
```

Comment:
Previously, the stats handler was manually created because message stats were not needed (since we didn't use streams). We started using streams (between Conduit and connectors), so we can now use this stats handler.

Reply:
We don't really collect metrics from the plugin using the gRPC stats handler. Regardless, I'm not sure why we had a custom stats handler in the first place tbh, even if we don't have streams, it doesn't really hurt if we register those stream stats collectors 👍